src/chk/chk.pb-c.c

```diff
@@ -1,12 +1,277 @@
+                     (const Chk__CheckReport *message)
+{
+  assert(message->base.descriptor == &chk__check_report__descriptor);
+  return protobuf_c_message_get_packed_size ((const ProtobufCMessage*)(message));
+}
+size_t chk__check_report__pack
+                     (const Chk__CheckReport *message,
+                      uint8_t       *out)
+{
+  assert(message->base.descriptor == &chk__check_report__descriptor);
+  return protobuf_c_message_pack ((const ProtobufCMessage*)message, out);
+}
+size_t chk__check_report__pack_to_buffer
+                     (const Chk__CheckReport *message,
+                      ProtobufCBuffer *buffer)
+{
+  assert(message->base.descriptor == &chk__check_report__descriptor);
+  return protobuf_c_message_pack_to_buffer ((const ProtobufCMessage*)message, buffer);
+}
+Chk__CheckReport *
+       chk__check_report__unpack
+                     (ProtobufCAllocator  *allocator,
+                      size_t               len,
+                      const uint8_t       *data)
+{
+  return (Chk__CheckReport *)
+     protobuf_c_message_unpack (&chk__check_report__descriptor,
+                                allocator, len, data);
+}
+void   chk__check_report__free_unpacked
+                     (Chk__CheckReport *message,
+                      ProtobufCAllocator *allocator)
+{
+  if(!message)
+    return;
+  assert(message->base.descriptor == &chk__check_report__descriptor);
+  protobuf_c_message_free_unpacked ((ProtobufCMessage*)message, allocator);
+}
+static const ProtobufCFieldDescriptor chk__check_report__field_descriptors[15] =
+{
+  {
+    "seq",
+    1,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_UINT64,
+    0,   /* quantifier_offset */
+    offsetof(Chk__CheckReport, seq),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "class",
+    2,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_ENUM,
+    0,   /* quantifier_offset */
+    offsetof(Chk__CheckReport, class_),
+    &chk__check_inconsist_class__descriptor,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "action",
+    3,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_ENUM,
+    0,   /* quantifier_offset */
+    offsetof(Chk__CheckReport, action),
+    &chk__check_inconsist_action__descriptor,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "repaired",
+    4,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_BOOL,
+    0,   /* quantifier_offset */
+    offsetof(Chk__CheckReport, repaired),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "rank",
+    5,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_UINT32,
+    0,   /* quantifier_offset */
+    offsetof(Chk__CheckReport, rank),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)